.vscode/settings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "cSpell.words": [
```

Comment:
The new .vscode/settings.json file adds spell check words but isn't mentioned in the PR description. Consider documenting this addition.